classes/PaymentModule.php

```diff
@@ -948,7 +949,8 @@ protected function createOrderFromCart(
         $debug,
         $order_status,
         $id_order_state,
-        $carrierId = null
+        $carrierId = null,
```

Comment:
Changing legacy core classes is always tricky, because we never know what we might break ^^
Especially in this one PaymentModule is a critical class for our e-commerce CMS

Besides it seems like your change in the prototype doesn't bring much since you always use an empty string as a value, so I would suggest not changing this method but instead set the new `Order::note` field with a default value `''` This would be a safer modification and it will have the same behaviour as you expect